src/views/Change.vue

```diff
@@ -50,31 +50,6 @@
 
       </div>
     </v-slide-x-reverse-transition>
-
-    <!-- Done-->
-    <v-fade-transition>
-      <div v-if="isSummaryMode && !showFeeSummary">
-        <header>
-          <h1>Review and Certify</h1>
-        </header>
-
-        <section class="mt-6">
-          You have deleted all fee-based changes and your company information has reverted to its
-          original state. If you made any non-fee changes such as updates to your Registered
-          Office Contact Information, please note that these changes have already been saved.
-        </section>
-
-        <v-btn
-          large
-          color="primary"
-          id="done-button"
```

Comment:
Please also delete the CSS for this button.

Reply:
Will do, thank you!

Reply:
Removed